prow/cmd/horologium/main_test.go

```diff
@@ -150,13 +164,26 @@ func TestSync(t *testing.T) {
 	}
 }
 
+func intP(i int) *int {
+	return &i
+}
+
+func intPToIntDefault1(i *int) int {
```

Comment:
Can we DRY this out with the actual prod code?

Reply:
Are you suggesting we use pjutil to create the prow jobs here and then just override the fields we want to set?  That is where the MaxConcurrency defaulting is currently happening, although it may shift a bit based on the changes on making MaxConcurrency a pointer in JobBase.